Sources/ApodiniDatabase/Context/TypeContainer.swift

```diff
@@ -184,5 +184,6 @@ extension TypeContainer: LosslessStringConvertible {
         // As query parameters are currently internally used a `.lightweight` and therefore not initialized using this init,
         // there is currently no mapping for this and will always be defaulted to `.noValue`.Will be added in the future.
         self = .noValue
```

Comment:
You can remove this assignment to `self` as the initializer will not terminate due to the `fatalError` below.